js/stdlib/ts/interfaces.ts

```diff
@@ -83,10 +83,15 @@ export interface Arith {
   bxor: (x: num, y: num) => BigNumber
   sqrt: (n1: num, n2: num) => BigNumber
   add256: (x: num, y: num) => BigNumber
+  safeAdd256: (x: num, y: num) => BigNumber
```

Comment:
Did you increment the JS version with this change? If so, you need to add the note in the Changelog about "changing the compiled interface"